src/Stratis.Bitcoin.Features.SmartContracts/PoS/SmartContractActivationProvider.cs

```diff
@@ -52,5 +60,17 @@ public SmartContractPosActivationProvider(Network network, NodeDeployments nodeD
                 }
             };
         }
+
+        /// <inheritdoc/>
+        public bool SkipRule(RuleContext context)
```

Comment:
Would be nice to see a few unit tests for this logic.

Reply:
Added.

Reply:
Thanks!